superset-frontend/src/views/datasetList/Modal.tsx

```diff
@@ -0,0 +1,97 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+import styled from '@superset-ui/style';
+import { Modal as BaseModal } from 'react-bootstrap';
+import { t } from '@superset-ui/translation';
+import Button from './Button';
+
+interface ModalProps {
+  children: React.ReactNode;
+  disableSave: boolean;
+  onHide: () => void;
+  onSave: () => void;
+  show: boolean;
+  title: React.ReactNode;
+}
+
+const StyledModal = styled(BaseModal)`
+  .modal-content {
+    border-radius: ${({ theme }) => theme.borderRadius}px;
+  }
+`;
```

Comment:
Can we use cascading styles in one root component instead of multiple styled modules? You are referencing bootstrap class names anyway. I made an argument about this earlier: https://github.com/apache-superset/superset-ui/pull/556#discussion_r434262576

This seems like another reason to not use custom components from `react-boostrap` (which is just fancy css class name builders).

@mistercrunch @rusackas @kristw in regard to the future of `react-boostrap`, if we really want to migrate away from it, I propose we start limiting its use to interaction-rich components only (modal, tooltip, etc); for pure CSS components, it's probably better to just use plain bootstrap classnames or create new self-contained styled components.

Reply:
I have moved all css to `Modal` component